src/spikeinterface/postprocessing/tests/test_correlograms.py

```diff
@@ -14,54 +12,47 @@
 from spikeinterface.postprocessing.tests.common_extension_tests import AnalyzerExtensionCommonTestSuite
 from spikeinterface.postprocessing import ComputeCorrelograms
 from spikeinterface.postprocessing.correlograms import compute_correlograms_on_sorting, _make_bins
+import pytest
 
 
-class ComputeCorrelogramsTest(AnalyzerExtensionCommonTestSuite, unittest.TestCase):
-    extension_class = ComputeCorrelograms
```

Comment:
I am a bit sad.
 I liked the idea that the commontest were aware the extension being tested.
This has could being used in test_extension no ?

Reply:
It's okay :D The tested extension is still passed to the `CommonTest`, it could be set as an attribute if required. I think this way made the pytest fixture generation easier, and allows better parameterizing over all the tests (so each tests displays on the output individually, rather than been run in a single test, in a loop in the original `test_extension()`). I think it makes the logic-flow easier to follow as a code reader. However if needed in future the extension could still be set as an attribute on the `CommonTest`.

```
    @pytest.mark.parametrize(
         "params",
         [
             dict(method="numpy"),
             dict(method="auto"),
             pytest.param(dict(method="numba"), marks=pytest.mark.skipif(not HAVE_NUMBA, reason="Numba not available")),
         ],
     )
     def test_extension(self, params):
         self.run_extension_tests(ComputeCorrelograms, params)
```